command/src/main/java/com/jagrosh/jdautilities/command/AnnotatedModuleCompiler.java

```diff
@@ -38,7 +38,7 @@
 public interface AnnotatedModuleCompiler
 {
     /**
-     * Compiles one or more {@link com.jagrosh.jdautilities.command.Command Command}s
+     * Compiles one or more {@link Command Command}s
```

Comment:
This breaks JDA Documentation standards, the original way this was done should have been maintained